components/homme/cmake/machineFiles/aurora-jit.cmake

```diff
@@ -0,0 +1,58 @@
+#module restore
```

Comment:
And "-jit"? I doubt it is "Just In Time"?

Reply:
it is "just in time" https://www.intel.com/content/www/us/en/docs/oneapi/optimization-guide-gpu/2023-0/just-in-time-compilation-in-sycl.html